automation/taskcluster/lib/tasks.py

```diff
@@ -494,42 +550,45 @@ def craft_raptor_task(self, signing_task_id, mozharness_task_id, variant):
                 } for worker_path, public_folder in (
                     ('artifacts', 'test'),
                     ('workspace/build/logs', 'logs'),
-                    ('workspace/build/blobber_upload_dir', 'test_info')
+                    ('workspace/build/blobber_upload_dir', 'test_info'),
                 )],
                 "command": [
                     "./test-linux.sh",
                     '--installer-url={}'.format(apk_location),
                     "--test-packages-url={}/{}/artifacts/public/build/target.test_packages.json".format(_DEFAULT_TASK_URL, mozharness_task_id),
-                    "--test=raptor-speedometer",
+                    "--test={}".format(test_name),
                     "--app=refbrow",
                     "--binary=org.mozilla.reference.browser",
                     "--activity=GeckoViewActivity",
                     "--download-symbols=ondemand"
-                ],
+                ] + extra_test_args,
                 "env": {
-                    "XPCOM_DEBUG_BREAK": "warn",
-                    "MOZ_NO_REMOTE": "1",
+                    "GECKO_HEAD_REPOSITORY": "https://hg.mozilla.org/mozilla-central",
+                    "GECKO_HEAD_REV": gecko_revision,
+                    "MOZ_AUTOMATION": "1",
                     "MOZ_HIDE_RESULTS_TABLE": "1",
-                    "TASKCLUSTER_WORKER_TYPE": 'proj-autophone/{}'.format(worker_type),
-                    "MOZHARNESS_URL": "{}/{}/artifacts/public/build/mozharness.zip".format(_DEFAULT_TASK_URL, mozharness_task_id),
+                    "MOZ_NO_REMOTE": "1",
+                    "MOZ_NODE_PATH": "/usr/local/bin/node",
+                    "MOZHARNESS_CONFIG": "raptor/android_hw_config.py",
                     "MOZHARNESS_SCRIPT": "raptor_script.py",
+                    "MOZHARNESS_URL": "{}/{}/artifacts/public/build/mozharness.zip".format(_DEFAULT_TASK_URL, mozharness_task_id),
+                    "MOZILLA_BUILD_URL": apk_location,
                     "NEED_XVFB": "false",
+                    "NO_FAIL_ON_TEST_ERRORS": "1",
```

Comment:
Are you sure that we don't want to fail the build if the test errors/fails to run?

Reply:
That's also what we have on m-c. What do you think @jmaher ?

Reply:
we have it on m-c for osx and android- although I admit not finding any code that uses that ENV var- lets leave it for now so we have parity.  I think a 2nd pass on cleaning this stuff up will be to have defined configs/environments in a single tree (i.e. m-c) and we can import those and have any changes in downstream trees (like r-b and fenix)- that should make the confusion less.